console.go

```diff
@@ -55,7 +58,13 @@ func (w ConsoleWriter) Write(p []byte) (n int, err error) {
 		if !w.NoColor {
 			lvlColor = levelColor(l)
 		}
-		level = strings.ToUpper(l)[0:4]
+		level = strings.ToUpper(l)
+		if LevelWidth > 0 {
+			for i := len(level); i < LevelWidth; i++ {
+				level += ` `
+			}
```

Comment:
What about: 
```go
if padding := LevelWidth - len(level); padding > 0 {
    level += strings.Repeat(" ", padding)
}
```

Reply:
Hi many thanks - I have updated the with the use of strings.Repeat but have found I still need a little more logic in order to handle the trimming of the column if the desired width is less than length of the level text.  Seen as we have worked out the padding it seems fastest to assume that if we don't pad then we trim the string to the desired length.  

I still need to treat the LevelWidth 0 as a special case trimming to 0 is the opposite of the desired behaviour of turning off the trimming/padding.